fedimintd/src/bin/main.rs

```diff
@@ -3,6 +3,7 @@ use fedimintd::Fedimintd;
 
 #[tokio::main]
 async fn main() -> anyhow::Result<()> {
+    println!("fedimintd:bin:main.rs -");
```

Comment:
No `println`, only `info!` and similar from `tracing`.

Reply:
This one is for my reference only....
I will use `info!` macro to print the version....
Though I have one question.... I have implemented the required line to get version But those lines don't provide me a function that will provide me the `Module/Consensus Version`
I was thinking may be I have to do something like this [LINK](https://github.com/fedimint/fedimint/blob/master/modules/fedimint-dummy-server/src/lib.rs#L102)
cause this will provide me the way to access version... Correct me If I'm wrong